beets/dbcore/db.py

```diff
@@ -32,6 +32,10 @@
 from .query import MatchQuery, NullSort, TrueQuery
 import six
 
+class AccessFileError(Exception):
+    """UI exception. Commands should throw this in order to display
+    nonrecoverable errors to the user.
```

Comment:
This looks like the description for UserError? Here's an alternative docstring:

> The SQLite database became inaccessible. This can happen when trying to read or write the database when, for example, the database file is deleted or otherwise disappears. There is probably no way to recover from this error.